yocto-dev/build_conf/local.conf

```diff
@@ -0,0 +1,41 @@
+MACHINE ??= 'ucm-imx93'
```

Comment:
Was the template for this taken from somewhere or did you write this from scratch? If the former is the case, you should add a reference to the original (repo link etc).

Reply:
This file is auto-generated when sourcing the Compulab's `compulab-setup-env` script for the first time as part of their README. This is appended to and we need to overwrite a newly generated version of this file on a fresh system, and similarly the `bblayers.conf`, with the appended Kimera related lines for our image. The ideal workflow to follow would be to implement a setup script to do this for us, as well as any other steps which can be automated.

Reply:
If we overwrite this every time we set up our environment on a new machine, then it should not be included in this repository at all because every time someone does setup, it will show up as a change. Please add a .gitignore file to this directory and include this file there.

Reply:
The files which are autogenerated are actually located in the `{BUILD_FOLDER_NAME}/conf` folder in the root of Compulab's setup directory.

Our `build_conf` folder does not mimic the structure and therefore will not show up as a change. This is simply a folder to store the configs we require, hence the need to overwrite the ones autogenerated.